docs/workforce-management-framework/topics/pro-using-demo-app.adoc

```diff
@@ -0,0 +1,110 @@
+= Using the demo application user interface
+
+The {WFM-RC-LongName} demo application consists of three components:
+
+- A mobile application for end users
+- A portal application for administrator users on a web browser
+- A cloud application for backend server functionality (No user interface)
+
+The demo application has demo users and demo data to show off the functionality of {WFM-RC-LongName}.
+The demo application can define business process or procedure via a custome Workflow and assign these workflows as jobs to
+individual workers as workorders. Workers can view their workorder via a mobile applicaion and administrators
+can managed workorders via the web portal.
+
+== Mobile application
+
+The mobile application is used by a end user to interact with workorders that have been assigned to them.
+login screen requires username and password. For a list of users see 
+link:https://github.com/feedhenry-raincatcher/raincatcher-core/blob/master/demo/server/src/modules/wfm-user/users.json[users.json],
+all users in the demo have a default password is `123`
+
+image::/shared/images/login.png[Login]
+
+Once logged in you will be presented with the workorders screen which will show the workorders assigned to the logged in user.
+In this example workorders are accident reports to be filled out at the seen of an accident.
+
+image::/shared/images/Workorder-mobile.png[Workorders]
+
+To log out click on the menu icon in the top left hand side of the screen this will open a sidebar with options to Logout or return
+to the Workorders screen.
+
+image::/shared/images/sidebar.png[Workorders]
+
+Return to the Workorders screen and click on an one of the workorders. The following screen will render
+
+image::/shared/images/workorder-overview.png[overview]
+
+Click on `BEGIN WORKFLOW` to proceed with data entry in the following `Accident Report`.
```

Comment:
Skip that part. We will need to explain that people will need to follow workflow but without specifying and explaining our "Accident flow workflow" this is just example - people may have different workflow selected that will invalidate this procedure.